docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
```

Comment:
Ensure that the volume my_media is correctly set up to handle media files, as this is crucial for managing media in a Docker environment.